vagrant/krd-vars.yml

```diff
@@ -8,8 +8,34 @@
 # http://www.apache.org/licenses/LICENSE-2.0
 ##############################################################################
 
-qat_driver_dest: "/tmp/qat"
-qat_driver_version: "1.7.l.4.3.0-00033"
-qat_driver_url: "https://01.org/sites/default/files/downloads/intelr-quickassist-technology/qat{{ qat_driver_version }}.tar.gz"
 # NOTE: Replace the `qat_envoy_dest` value with the current absolute path
 qat_envoy_dest: "/home/vagrant/kubernetes-qat-envoy/"
+qat_base_version: "1.7.l.4.4.0-00023"
+#qat_base_version: "1.7.l.4.3.0-00033"
+qat_driver_version: "{{ qat_base_version }}"
+qat_plugin_version: "{{ qat_base_version }}"
+qat_driver:
+  dest: "/tmp/qat"
+  url: "https://01.org/sites/default/files/downloads/qat{{ qat_driver_version }}.tar.gz"
+  #url: "https://01.org/sites/default/files/downloads/intelr-quickassist-technology/qat{{ qat_driver_version }}.tar.gz"
+qat_plugin:
+  dest: "{{ qat_envoy_dest }}/qat{{ qat_plugin_version }}.tar.gz"
+  url: "https://01.org/sites/default/files/downloads/qat{{ qat_driver_version }}.tar.gz"
+  #url: "https://01.org/sites/default/files/downloads/intelr-quickassist-technology/qat{{ qat_plugin_version }}.tar.gz"
+kata_dest: "/tmp/kata"
+intel_plugin_url:
+  kata:
+    img_name: "intel-qat-plugin"
+    deployment_files:
+      - "qat_plugin/qat_plugin.yaml"
+      - "qat_plugin/qat_plugin_default_configmap.yaml"
+    url: "https://github.com/intel/intel-device-plugins-for-kubernetes"
+    version: "b2100f2735f0a8b759d5a34f2ccc6f80f422d499"
+  runc:
+    img_name: "intel-qat2-plugin"
```

Comment:
everything uses intel-qat-plugin now. with ```-mode kernel``` you get what qat2-plugin did. (see the deployment yamls in the upstream repo)

Reply:
Fixed on 8d77c25655aa6c21cbdfe98ba3dd242e90cef454